tests/Unit/BasicTest.php

```diff
@@ -185,4 +223,22 @@ public function can_pass_in_our_own_exception()
 
         $this->assertInstanceOf(Result::class, $result);
     }
+
+    /** @test */
+    public function it_does_not_throw_for_non_exceptions_when_protections_are_bypassed()
```

Comment:
@aarondfrancis I think I found a small bug here. Seems non-exception values were been thrown when protection was disabled.

Reply:
Ha I *just* got bit by that! Thank you